src/cairoWriter.ts

```diff
@@ -86,6 +86,17 @@ import { isCairoConstant } from './utils/utils';
 
 const INDENT = ' '.repeat(4);
 
+function getDocumentation(
+  documentation: string | StructuredDocumentation | undefined,
+  writer: ASTWriter,
+): string {
+  return typeof documentation !== 'undefined'
```

Comment:
This initial condition can be simplified to `documentation !== undefined`, no need for the `typeof`

Reply:
Done